src/gui/folderman.cpp

```diff
@@ -817,7 +817,7 @@ void FolderMan::startScheduledSyncSoon()
         return;
     }
 
-    seconds delay;
+    seconds delay { 2s }; // Startup, if _lastSyncFolder is still empty.
```

Comment:
Nitpic: you can save that comment by moving it into the else case :D

Reply:
Anyhow it should default to 1s why do we need the 2s?

Reply:
`delay = qBound(1s, delay - sinceLastSync, 60s);`

Reply:
I hate uninitialized variables as they are always a source of errors, like here. 
Default can be 1s, I dont care.